lib/ansible/module_utils/common/collections.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c), Sviatoslav Sydorenko <ssydoren@redhat.com> 2018
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+"""Collection of low-level utility functions."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+# Note: When getting Sequence from collections, it matches with strings.  If
+# this matters, make sure to check for strings before checking for sequencetype
+try:
+    """Python 3.3+ branch."""
+    from collections.abc import KeysView, Sequence
+except ImportError:
+    """Use old lib location under 2.6-3.2."""
+    from collections import KeysView, Sequence
+
+SEQUENCETYPE = frozenset, KeysView, Sequence
```

Comment:
We can remove the SEQUENCETYPE setting from here now that it is not used.

Reply:
Isn't it a thing we might want to expose from collections module?

Reply:
I don't think we do.  It's a wierd hybrid of Sequence and random other objects.  I think we'd want to leave it in basic.py as backwards compatibility but not bring it into our new API.